lib/utils/containsString.js

```diff
@@ -1,6 +1,8 @@
 /* @flow */
 'use strict';
 
+const _ = require('lodash');
```

Comment:
You're were removing Lodash... :)

Reply:
We can't remove it completely :) No need to reinvent the wheel, unless you want us to maintain our own set of utils :)